v2/api/insights/v1api20180501preview/storage/structure.txt

```diff
@@ -2,93 +2,108 @@
 github.com/Azure/azure-service-operator/v2/api/insights/v1api20180501preview/storage
 ├── APIVersion: Enum (1 value)
 │   └── "2018-05-01-preview"
-└── Webtest: Resource
```

Comment:
minor: @theunrepentantgeek, it would be nice if the format for this file didn't change so drastically when additions were made.

Maybe pure whitespace/indentation is enough and we can drop the `|`? Or maybe we could just render `|` anyway even if there's nothing below so that when something is added the change doesn't edit all the above lines too?

LMK if you think worth filing an issue over it.